### ISSUE TITLE
Unify napari_omero and omero napari view

### ISSUE THREAD

**Issue by joshmoore** [tlambert03/napari-omero#21]
I keep forgetting to use `omero napari view Image:1` rather than `napari_omero Image:1` and then end up wondering where the "Save ROIs to OMERO` button is.

**Comment by tlambert03** [comment]
agreed... need to get back to those ROI saving functions.  I _think_ we're gonna want (even a simple version of) #16 to be merged for that

**Comment by psobolewskiPhD** [comment]
#100 partially addresses this, by making it possible to upload via widget when opening an image from the browser or using the reader. We can leave open for the time being, but it might be low priority.
